src/plugins/action_create_user_from_application/src/createUserFromApplication.ts

```diff
@@ -0,0 +1,64 @@
+import { BasicObject } from '@openmsupply/expression-evaluator/lib/types'
+
+const responseMap = {
+  // Change these when question codes change
+  username: 'Q3',
+  first_name: 'Q1',
+  last_name: 'Q2',
+  email: 'Q4',
+  password_hash: 'Q5',
+}
```

Comment:
I'm trying to understand how would this work when we allow different templates created.
Would we have one action per template to would store those fields hardcoded?
There would be a way to send which fields { title, codes } to use? 
Thinking on when we allow people to create a template or even when we have to maintain more than one template, how do we keep those up-to-date?

These are only some concerns, I don't think we need to make any changes to this PR - we can add new issues if we see any improvements that can be done and - maybe add some instructions to the wiki about how to create and maintain the actions (specially regarding the output parameters).

Reply:
This Action is very much meant to be specifically tied to the current UserRegistration template. As I said in the original post, it just makes writing the lookup queries much simpler, but the generic `createUser` Action works fine too -- it just requires a lot more fiddling in the template config to extract the values required for input.

Since the User Registration template is going to be a "core" template, with probably not a lot of variation, I thought it made sense to have a dedicated handler for it.